src/utils/init.js

```diff
@@ -108,7 +115,7 @@ function initInfo() {
 function updateUserKey(newUserKey, merge = false) {
   defaultClient.updateUserKey(newUserKey, merge).then(() => {
     initState.userKey = newUserKey;
-    if (initState.tracking === 'cookie') {
+    if (initState.tracking === 'cookie' && typeof window === 'object') {
```

Comment:
what if instead, we create utility functions for accessing window and document that check if they exist first? instead of having all of these window checks across the codebase